data/LRWholeGenomeSingleSample/TestWithChrM.json

```diff
@@ -10,5 +10,8 @@
     "LRWholeGenomeSingleSample.ref_fasta":     "gs://broad-dsde-methods-long-reads/resources/references/grch38/Homo_sapiens_assembly38.fasta",
     "LRWholeGenomeSingleSample.ref_fasta_fai": "gs://broad-dsde-methods-long-reads/resources/references/grch38/Homo_sapiens_assembly38.fasta.fai",
     "LRWholeGenomeSingleSample.ref_dict":      "gs://broad-dsde-methods-long-reads/resources/references/grch38/Homo_sapiens_assembly38.dict",
+
+    "LRWholeGenomeSingleSample.tandem_repeat_bed": "gs://broad-dsde-methods-long-reads/2019-Aug-PacBio-Nature-Biotech-Paper/human_GRCh38_no_alt_analysis_set.trf.bed",
```

Comment:
We should copy the tandem repeat finder file over to gs://broad-dsde-methods-long-reads/resources/references/grch38/ .

Reply:
I've thought about that, but I'm not sure how we should provide information on how each file was produced.

A real, related situation that I face: while test running this subworkflow before issuing this PR, is that the reference `hs37d5.fasta` does not work well with `gs://broad-dsde-methods-long-reads/2019-Aug-PacBio-Nature-Biotech-Paper/human_hs37d5.trf.bed` for PBSV. The reason is that `hs37d5.fasta` contains the decoy contig at the end (hence 86 contigs), while the SVSIG file produced (by `pbsv discover`) with that TRF annotation requests the reference file  in the following `pbsv call` stage to have exactly 85 contigs in the reference. Annoying, but reality...

The point is that we should have a way of keeping track of how each reference related annotation is produced (if downloaded, note source; if custom, note program, parameters and inputs used).
What do you think?